components/core/SearchModal.js

```diff
@@ -113,7 +119,14 @@ const STYLES_PROFILE_IMAGE = css`
 const UserPreview = ({ user }) => {
   return (
     <div>
-      <div css={STYLES_PROFILE_IMAGE} style={{ backgroundImage: `url('${user.data.photo}')` }} />
+      <div css={STYLES_PROFILE_IMAGE}>
+        <ProfilePhoto 
+          photo={user.data.photo}
```

Comment:
And you forgot to update this to the new format where you pass in user instead of photo and id